src/en/authors-charm-hooks.md

```diff
@@ -58,36 +38,46 @@ one-time setup operations only.
   - immediately after "install"
   - immediately after "upgrade-charm"
   - at least once when the unit agent is restarted (but, if the unit is in an
-[error state](./authors-hook-errors.html), it won't be run until after the
-error state is cleared).
-  - after changing charm configuration using a command line interface
+    [error state](./authors-hook-errors.html), it won't be run until after the
+    error state is cleared).
+  - after changing charm configuration using the GUI or command line interface
 
 It cannot assume that the software has already been started; it should not start
 stopped software, but should (if appropriate) restart running software to take
 configuration changes into account.
 
+### install
+
+`install` is run at the beginning of a charm lifecycle. The hook should be used
+to perform one-time setup operations, such as installing prerequisite software
+that will not change with configuration changes. Like all hooks this must be
+idempotent because there are scenarios where this hook can be run more than
```

Comment:
Pedantry. +1 to calling out idempotency, but its only ever invoked again if the charm logic itself specifically invokes the install hook, which would have to be done by the charm author.  Again, pedantic, but thought it was worth mentioning.